subsys/net/lib/nrf_cloud/src/nrf_cloud_transport.c

```diff
@@ -75,6 +75,10 @@ BUILD_ASSERT(IMEI_CLIENT_ID_LEN <= NRF_CLOUD_CLIENT_ID_MAX_LEN,
 #define NCT_UPDATE_TOPIC AWS "%s/shadow/update"
 #define NCT_SHADOW_GET AWS "%s/shadow/get"
 
+/* Buffers to hold stage and tenant strings. */
+static char stage[8];
+static char tenant[40];
```

Comment:
would it be useful to have a define for tenant id length and reference it in the get function comment